Smart Goods/Common/Loadable.swift

```diff
@@ -7,24 +7,16 @@
 
 import Foundation
 
-public enum Loadable<Item> where Item: Codable, Item: Equatable {
+public enum Loadable<Item>: Equatable where Item: Codable, Item: Equatable {
     case loading
     case loaded(Item)
     case error(HTTPError)
     case none
-}
 
-extension Loadable: Equatable {
```

Comment:
one must be a Swift expert to realise that this extension is not necessary

Reply:
ha ha ha